<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkinSelector</class>
 <widget class="QMainWindow" name="SkinSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkinSelector</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="skinListView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>321</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>561</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>191</y>
      <width>461</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Skin folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="skinPathEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="skinPathButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>301</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Regular skins</string>
    </property>
   </widget>
   <widget class="QListView" name="pianoSkinListView">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>521</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="pianoCheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>491</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Piano display Skin</string>
    </property>
   </widget>
   <widget class="QListView" name="subSkinListView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>321</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="previewGraphicView">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>221</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>500</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>status label</string>
    </property>
   </widget>
   <widget class="InputSourceSelector" name="inputSourceSelector" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>611</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>InputSourceSelector</class>
   <extends>QWidget</extends>
   <header>inputsourceselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
